libwd.map

```diff
@@ -0,0 +1,39 @@
+UADK_2.0 {
```

Comment:
I prefer to follow the naming policy as "LIBWD_2.0".

https://www.gnu.org/software/gnulib/manual/html_node/LD-Version-Scripts.html

Reply:
We mainly reflect the UADK framework now, so the corresponding lib library interface version name is declared as UADK_2.0